<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminPage</class>
 <widget class="QMainWindow" name="AdminPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>301</width>
      <height>219</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="UserDetails_verticalLayout">
     <item>
      <layout class="QVBoxLayout" name="Username_verticalLayout">
       <item>
        <widget class="QLabel" name="UserName_label">
         <property name="text">
          <string>UserName</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="UserNameDetails_label">
         <property name="text">
          <string>UserName Details </string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="FirstName_verticalLayout">
       <item>
        <widget class="QLabel" name="FName_label">
         <property name="text">
          <string>First Name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="FNameDetails_label">
         <property name="text">
          <string>First Name Details</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="Lname_verticalLayout">
       <item>
        <widget class="QLabel" name="LName_label">
         <property name="text">
          <string>Last Name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="LNameDetails_label">
         <property name="text">
          <string>Last Name Details</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="Email_verticalLayout">
       <item>
        <widget class="QLabel" name="Email_label">
         <property name="text">
          <string>Email</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="EmailDetails_label">
         <property name="text">
          <string>Email Details</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QListWidget" name="Tasks_listWidget">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>40</y>
      <width>421</width>
      <height>511</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>0</y>
      <width>421</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="Search_horizontalLayout">
     <item>
      <widget class="QPushButton" name="Search_pushButton">
       <property name="styleSheet">
        <string notr="true">border-color: rgba(255, 255, 255, 0);</string>
       </property>
       <property name="text">
        <string>Search</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="Search_lineEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="AddUser_pushButton">
       <property name="text">
        <string>Add User</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Delete_pushButton">
       <property name="text">
        <string>Delete User</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="CloseApp_pushButton">
       <property name="text">
        <string>Close App</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
